liferay-tango/liferay-portal
# LPS-128385 As a site admin, I want to configure PageSpeed from Site Settings

This adds support for fully configuring PageSpeed from site settings using the new simplified approach based on Configuration Classes.

**How to test it**

1. Go to Site Settings -> Pages -> Google PageSpeed Insights.
2. Set the API Key from there.
3. Everything should work.
